src/main/webapp/video.js

```diff
@@ -45,3 +43,31 @@ function onYouTubeIframeAPIReady() {
 function onPlayerReady(event) {
   event.target.playVideo();
 }
+
+/** Starts timer if {@code currState} is playing, stops otherwise. */
```

Comment:
please rename currState to currentState.

It is recommended to not use abbreviations, see https://docs.google.com/document/d/1_ldClEZLsI1cMpQl_BfeIlkjTsE2IiJ2PgvhABfO6vs/edit#heading=h.wx9q7rdncnvo

Reply:
fixed